### PR TITLE
Project 1: setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by Debas-31**
# Milestone 1: Setup and mobile version skeleton
>In this milestone of the project: 
- Added Header 
- Added Headline-section
- Added linters setup

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    <main class="container">
+      <header>
+        <nav class="navbar" id="nav">
+            <a href="#" class="logo">Dbus-G'</a>
+            <div class="menu">
+                <a href="#" class="humberger"><i class="fas fa-bars"></i></a>
+            </div>
+        </nav>
+      </header>
+      <section class="headline-section">
```
Comment (TedLivist) [inline index.html:24]: - [ ] Please ensure that your **headline-section** is aligned with **`flexbox`**. In your CSS, that does not seem to be the case.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+          <div class="headline-content">
+              <h1>Debas G'ngus</h1>
+              <h2>Software Development</h2>
+              <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me. </p>
+          </div>
+          <img src="./image/horizontal-line.png" alt="horizontal line image">
+          <div class="skill">
+              <ul class="lists"><li>Language</li>
```
Comment (TedLivist) [inline index.html:32]: - [ ] Please indent the **`li`** tag as it is a child of **`ul`**

File: index.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+          <div class="skill">
+              <ul class="lists"><li>Language</li> 
+                  <li>Javascript</li>
+                  <li>Ruby</li>
+                  <li>Html</li>
+                  <li>Css</li>
+              </ul>
+              <ul class="lists">
+                  <li>Frameworks</li>
+                <li>Bootsrap</li>
```
Comment (TedLivist) [inline index.html:40]: - [ ] In keeping with [HTML and CSS practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) please fix your indentation here

File: index.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+                <li>Skills</li>
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlav</li>
+                <li>Terminal</li>
+            </ul>
+          </div>
+          <img src="./image/horizontal-line.png" alt="horizontal line image">
+      </section>
```
Comment (TedLivist) [inline index.html:56]: - [ ] There seems to be an issue with the background image of your headline. According to the template, the circles should stretch across the screen but in your design that isn't the case. Please fix this. Look at the image below:

debas-headline

- [ ] Going by the template, you are missing the headline icons. Please add them. **Note:** they should be wrapped in a `<ul>` tag with each of them being a link wrapped in a `<li>`

debas-icons

File: index.html [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Debas Portfolio | Dbus</title>
+</head>
+<body>
+    <main class="container">
+      <header>
```
Comment (TedLivist) [inline index.html:16]: - [ ] **[OPTIONAL]** Going by conventional, the **`main`** tag should not wrap your **`header`**. They are semantic tags with different purposes. The `<header>` can be a stand-alone element, while the `<main>` tag can wrap other parts of your main sections

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️ 

Hi @Debas-31,

Good job so far!
✔️ Your linters are properly set up
✔️ Nice design
However, there are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alexoid1 — APPROVED** [review]
# STATUS: APPROVED!!!
## Hi @Debas-31  Great job! 
## Congratulations :tada: Thanks for implementing changes suggested.
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada:

# OPTIONAL

brent-rambo-thumbs-up


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**